Mathlib/Order/SupClosed.lean

```diff
@@ -478,3 +479,69 @@ def SemilatticeInf.toCompleteSemilatticeInf [SemilatticeInf α] (sInf : Set α 
   sInf := fun s => sInf (infClosure s)
   sInf_le _ _ ha := (h _ infClosed_infClosure).1 <| subset_infClosure ha
   le_sInf s a ha := (le_isGLB_iff <| h _ infClosed_infClosure).2 <| by rwa [lowerBounds_infClosure]
+
+
+section ConditionallyCompleteLattice
+variable [ConditionallyCompleteLattice α] {f : ι → α} {s t : Set α}
+
+lemma SupClosed.iSup_mem_of_nonempty [Finite ι] [Nonempty ι] (hs : SupClosed s)
```

Comment:
```suggestion
lemma SupClosed.ciSup_mem [Finite ι] [Nonempty ι] (hs : SupClosed s)
```

Reply:
I disagree with this name. This lemma is also valid (and useful) for complete lattices, and the current name helps disambiguate from the other `iSup_mem` lemma.